### PR TITLE
Publisher bearer token

### PR DESCRIPTION
**Pull request by njseeto**
Since the bearer tokens are have different names in the two different clusters (K8S and EKS), we need a way to be able to choose the correct token respective of the cluster we are deploying to.

Co-authored-by: Tomas Destefi <tomas.destefi@digital.justice.gov.uk>
Co-authored-by: Brendan Butler <brendan.butler@digital.justice.gov.uk>

### CODE REVIEW

File: bin/deploy [hunk 1]
```diff
@@ -157,6 +157,18 @@ fi
...
 echo
 
+if [[ $application_name == 'fb-publisher' ]]; then
+  # The clusters (K8S and EKS) have different names for their bearer tokens
+  # So these tokens have been set in the deployment pipeline
+  # Each token is different also per environment (test and live) so
+  # it needs this dynamic grep for K8S_BEARER_TOKEN_TEST and
+  # K8S_BEARER_TOKEN_LIVE
+  bearer_token_env_var_name="K8S_BEARER_TOKEN_${platform_environment}"
+  bearer_token_env_var_name=$(echo $bearer_token_env_var_name | tr '-' '_' | tr [a-z] [A-Z])
```
Comment (brenetic) [inline bin/deploy]: is the `tr '-' '_'` just in case we accidentally add an envar with a `-`?

Comment (tomas-stefano) [reply bin/deploy]: I think I copied from other scripts. Should I removed it?

Comment (brenetic) [reply bin/deploy]: yeah i think we can remove it. publisher (and editor) will only ever be test or live so there won't be a need to swap the `-` 🤞

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**